maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/pref/MavenArchetypePreferenceInitializer.java

```diff
@@ -70,84 +70,6 @@ public void initializeDefaultPreferences() {
 
 		// new project template archetypes
 
-		defaultPrefs.put(
-			LiferayMavenCore.PREF_ARCHETYPE_PROJECT_TEMPLATE_ACTIVATOR,
```

Comment:
Can we remove these preference keys from the `LiferayMavenCore` class now as well?

Reply:
yes, I will remove them and resend